src/gui/socketapi/socketapi.cpp

```diff
@@ -1116,11 +1146,12 @@ void SocketApi::command_GET_STRINGS(const QString &argument, SocketListener *lis
     listener->sendMessage(QString("GET_STRINGS:END"));
 }
 
-void SocketApi::sendSharingContextMenuOptions(const FileData &fileData, SocketListener *listener, bool enabled)
+void SocketApi::sendSharingContextMenuOptions(const FileData &fileData, SocketListener *listener, bool isForE2eeItem, bool isForRootE2eeFolder)
```

Comment:
could we get enum instead of bool
using bool in that way makes it very hard to understand what will happen when reading code like this
`sendSharingContextMenuOptions(file, listener, true, false);`
or
`sendSharingContextMenuOptions(file, listener, false, false);`

Reply:
@mgallien I do not fully agree because your example is not used in my calls: `sendSharingContextMenuOptions(fileData, listener, isE2eEncryptedPath, isE2eEncryptedRootFolder);`., but let's have it